empire/config_manager.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import os
+import shutil
+import yaml
+
+from pathlib import Path
+
+log = logging.getLogger(__name__)
+
+user_home = Path.home()
+SOURCE_CONFIG_CLIENT = Path("empire/client/config.yaml")
+SOURCE_CONFIG_SERVER = Path("empire/server/config.yaml")
+CONFIG_DIR = user_home / ".empire"
+CONFIG_CLIENT_PATH = CONFIG_DIR / "client" / "config.yaml"
+CONFIG_SERVER_PATH = CONFIG_DIR / "server" / "config.yaml"
+
+def config_init():
+    CONFIG_CLIENT_PATH.parent.mkdir(parents=True, exist_ok=True)
+    CONFIG_SERVER_PATH.parent.mkdir(parents=True, exist_ok=True)
+    
+    if not CONFIG_CLIENT_PATH.exists():
+        shutil.copy(SOURCE_CONFIG_CLIENT, CONFIG_CLIENT_PATH)
+        log.info(f"Copied {SOURCE_CONFIG_CLIENT} to {CONFIG_CLIENT_PATH}")
+    else:
+        log.info(f"{CONFIG_CLIENT_PATH} already exists.")
+
+    if not CONFIG_SERVER_PATH.exists():
+        shutil.copy(SOURCE_CONFIG_SERVER, CONFIG_SERVER_PATH)
+        log.info(f"Copied {SOURCE_CONFIG_SERVER} to {CONFIG_SERVER_PATH}")
+    else:
+        log.info(f"{CONFIG_SERVER_PATH} already exists.")
```

Comment:
Potential issue I see here is a user can pass a custom config location via `--config` but this assumes the config is always in the repo directory. Is there a way to support `--config`?

Reply:
No @vinnybod . It is not a potential issue because:
1- When you run Empire the first time, it runs `config_init()` and copies the two config files to user home config dir if they don't exist... BUT...
2- [client config](https://github.com/BC-SECURITY/Empire/blob/4361ca75db1ca6c3ce338c6968f565162d228fe0/empire/client/src/EmpireCliConfig.py#L11) and [server config](https://github.com/BC-SECURITY/Empire/blob/4361ca75db1ca6c3ce338c6968f565162d228fe0/empire/server/core/config.py#L130) say:
  * if user provides `--config`, load config from the provided location. Then...
  * if the provided config is empty (that means either you passed NO config file or you didn't use `--config` parameter), use the default client config file (the one we copied on HOME